packages/schema/src/utils/strip-quotes.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Strip leading/trailing quotes from a string
+ */
+export function stripQuotes(value?: string | null): string | null {
+	if (value == null) {
```

Comment:
```suggestion
	if (!value) {
```

This would also solve the above issue.

Reply:
Lets not accept undefined in that function 👍🏻

Reply:
Sounds good... But can we at least change it to a `value === null`? 😅